developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -3,8 +3,8 @@
 :toc: right
 :toclevels: 5
 
-== Installing the latest development code
-Foreman uses Bundler to install dependencies and get up and running. This is the preferred way to get Foreman if you want to benefit from the latest improvements. By using the git repository you can also upgrade more easily. You will need to have Bundler installed manually for now (check your distribution repositories, or install it via rubygems).
+== Good to Notice
+Foreman uses Bundler to install dependencies and get up and running. This is the preferred way to get Foreman if you want to benefit from the latest improvements. By using the git repository you can also upgrade more easily. You will need to have Bundler installed manually for now (this is taken care of during the 'setting the repository' step).
```

Comment:
Rethinking about it, I am not sure we need this text, at least I never installed `bundler` manually from git...
and if we do want to keep it, it can move into the `setting the repository` section.

Reply:
I think we should at least mention it, especially because this tool is used in Foreman, so I think it would be better like you suggested moving this notice to the setting the repository section (and changing the sentence in the parentheses)
Maybe we should also drop the latest section: "You will need to have Bundler installed manually for now", mainly for the reason we don't really need to install it manually

What do you think @Ron-Lavi ?